file-attribute-caching/src/test/kotlin/com/pkware/filesystem/attributecaching/AttributeCachingFileSystemTests.kt

```diff
@@ -350,27 +358,37 @@ class AttributeCachingFileSystemTests {
             // verify that attribute is "right" type returned from the provider
             assertThat(attributesMap).isInstanceOf(MutableMap::class.java)
             assertThat(attributesMap.size).isEqualTo(expectedMapSize)
-            // verify attribute map keys do not contain the attribute view qualifier (attributeViewName)
+            // verify attribute map keys do not contain the attribute view qualifier "attributeViewName:"
             for (entry in attributesMap) {
-                assertThat(entry.key).doesNotContain(attributeViewName)
+                assertThat(entry.key).doesNotContain("$attributeViewName:")
             }
         }
     }
 
-    @ParameterizedTest
-    @MethodSource("posixFileSystems")
-    fun `set posix attributes for path`(fileSystem: () -> FileSystem) = fileSystem().use { jimfs ->
-        AttributeCachingFileSystem.wrapping(jimfs).use {
+    @Test
+    @DisabledOnOs(OS.WINDOWS)
+    fun `set and read posix attributes for path`() = AttributeCachingFileSystem.wrapping(FileSystems.getDefault()).use {
+        // get filesystem attribute caching path
+        val javaTmpDir = it.getPath(System.getProperty("java.io.tmpdir"))
```

Comment:
so the issue here was that / calls resolve with a string and in those cases we were not returning caching paths, i added tests for those and cleared out these asserts

there was a second issue where the final caching path never gets initialized after the file is made so I forced it to become initialized and added asserts around that because the acl/dos tests depend on the cache being setup properly